### ISSUE TITLE
New Skype app for Linux users

### ISSUE THREAD

**Issue by gogi983** [chrisss404/gnome-shell-ext-SkypeNotification#67]
Hi there, I come back to you after long time as Skype has started providing a new web-based version of the app, and the extension of course doesn't work with this... Can you update the code and make it work with this new version please? 
By the way it is named "skypeforlinux alpha version" and they have even opened a repo with .deb and .rpm packages. 

Regards.

**Comment by chrisss404** [comment]
Hi, thanks for letting me know. I think it is currently not possible, however, I have asked for their plans in regard to this issue.

Let's see what they answer: https://community.skype.com/t5/Linux/Skype-for-Linux-Alpha-and-calling-on-Chrome-amp-Chromebooks/m-p/4460324/highlight/true#M15621

Best regards

**Comment by gogi983** [comment]
Hi,
Yep I saw your message onf their thread, but as you say I think too that it is currently not possible as it seems that they have disabled API support for third party apps. I read this somewhere but I can't find the link right now.
We can only expect an answer from them for the moment, by the way thanks for all the efforts you provide.

Best regards.

**Comment by chrisss404** [comment]
Just found the following thread, doesn't seem like they'll add an API.
https://community.skype.com/t5/Linux/Where-are-the-advanced-settings-for-Skype-Linux-Alpha/td-p/4456993

**Comment by gogi983** [comment]
Yep, that's the thread I mentioned in my previous answer and which I was looking for...
Btw, it looks like this is going to be an another fu**ing web app and not a desktop one, like what Twitter did to TweetDeck some months ago... They willingly killed the desktop app so users who still want to use TweetDeck have to go through the URL...
These proprietary apps owners are really scum bags, that's why I have gone far away from everything that's proprietary...

But I'm going to let them a message on the thread above, with copy/paste of the message you wrote on the other one, as they didn't answer yet...
